### PR TITLE
Fix crash when parsing empty files

### PR DESCRIPTION
**Pull request by pleonex**
The script crashes when the user tried to give as input an empty file.
I have also fix some links in the *Messages.md* file and add two command line commands to increase verbosity from the command line in Windows and with tcsh.

### CODE REVIEW

File: logparser/devices/inputdevices.py [hunk 1]
```diff
@@ -124,6 +124,9 @@ def print_progress(self, threshold=0, decimals=1, barLength=100):
         # Based on @Greenstick's reply (https://stackoverflow.com/a/34325723)
         iteration = self.stream.tell()
         total = self.file_size
+        if total == 0:
+            return
+
         progress = 100.0 * iteration / total
         if self.progress > 0 and progress - self.progress < threshold:
```
Comment (iblancasa) [inline logparser/devices/inputdevices.py]:
```python
 if self.progress > 0
```

Should be 
```python
 if self.progress
```

File: logparser/devices/inputdevices.py [hunk 2]
```diff
@@ -124,6 +124,9 @@ def print_progress(self, threshold=0, decimals=1, barLength=100):
         # Based on @Greenstick's reply (https://stackoverflow.com/a/34325723)
         iteration = self.stream.tell()
         total = self.file_size
+        if total == 0:
```
Comment (iblancasa) [inline logparser/devices/inputdevices.py:127]: I think that this should be:
```python
if not total:
```

But this is not enough intuitive. What do you think?

**Review by iblancasa — CHANGES_REQUESTED** [review]

File: logparser/devices/inputdevices.py (hunk 2 above)
Comment (pleonex) [reply logparser/devices/inputdevices.py:127]: I think that reading "if not total" is not intuitive. It make sense to compare against 0.

Comment (iblancasa) [reply logparser/devices/inputdevices.py:127]: I agree

**Review by iblancasa — APPROVED** [review]

**Pull request merged**